sympy/integrals/laplace.py

```diff
@@ -1132,6 +1132,7 @@ def _frac(f, s):
     _ILT_rules = [
         (a/s, a, S.true, same, 1),
         (b*(s+a)**(-c), t**(c-1)*exp(-a*t)/gamma(c), c>0, same, 1),
+        (1/(s**2+a**2)**2, (sin(a*t) - a*t*cos(a*t))/(2*a**3), S.true, same, 1)
```

Comment:
I think that this additional rational function rule is added silently though.

Reply:
Ah ... almost, I announced it in #24561 but forgot to mention it again here. This is one interesting case that `apart` does not handle.